### ISSUE TITLE
Question: Release 0.0.4 

### ISSUE THREAD

**Issue by FUSAKLA** [camptocamp/grafana-prometheus-alertmanager-datasource#18]
Hi, at first thank's for the plugin! 

I'm wondering if you plan releasing the `0.0.4` version on Grafna.com so it can be installed using the

```bash
grafana-cli plugins install camptocamp-prometheus-alertmanager-datasource
```

That would be really great. I'm looking forward the manual severity levels.

**Comment by cryptobioz** [comment]
A PR is open to add it to the catalog: https://github.com/grafana/grafana-plugin-repository/pull/253.

**Comment by cryptobioz** [comment]
The PR has been merged, the version 0.0.4 is now available to download.

**Comment by FUSAKLA** [comment]
Yep I've been watching it, thanks once more!
